apps/transport/lib/transport_web/controllers/aoms.ex

```diff
@@ -72,43 +101,48 @@ defmodule TransportWeb.AOMSController do
       )
       |> Repo.all()
 
-    datasets_by_aom_id = datasets |> Enum.group_by(& &1.aom_id)
-    datasets_by_dataset_id = datasets |> Enum.group_by(& &1.dataset_id)
+    gtfs_datasets_by_aom_id = gtfs_datasets |> Enum.group_by(& &1.aom_id)
+    gtfs_dataset_by_dataset_id = gtfs_datasets |> Map.new(&{&1.dataset_id, &1})
+    {gtfs_datasets_by_aom_id, gtfs_dataset_by_dataset_id}
+  end
 
-    aggregated_datasets =
+  defp get_aggregated_datasets(gtfs_dataset_by_dataset_id) do
+    aggregated_datasets_in_db =
       AOM
       |> join(:inner, [aom], d in assoc(aom, :legal_owners_dataset), as: :legal_owners_dataset)
       |> where(
         [legal_owners_dataset: d],
         d.id in subquery(
           Dataset.base_query()
           |> join(:inner, [dataset: d], aom in assoc(d, :legal_owners_aom), as: :aom)
+          |> join(:inner, [dataset: d], resource in assoc(d, :resources), as: :resource)
           |> group_by([dataset: d], d.id)
           |> having([aom: a], count(a.id) >= 2)
+          |> where([resource: r], r.format in ["GTFS", "NeTex"])
```

Comment:
```suggestion
          |> where([resource: r], r.format in ["GTFS", "NeTEx"])
```

Attention à la casse de ce format